test/system/150-login.bats

```diff
@@ -5,10 +5,14 @@
 
 load helpers
 load helpers.network
+load helpers.systemd
 
 ###############################################################################
 # BEGIN one-time envariable setup
 
+# For the auto-update test
+SNAME_FILE=$BATS_TMPDIR/services
```

Comment:
`$PODMAN_TMPDIR` please

Reply:
That's copied from the auto-update tests.  Shall I change it there as well?

Reply:
Might as well. Yes please.

Reply:
ARGH. Cannot be `$PODMAN_TMPDIR` because that's not actually defined at that point in time. Hold on, don't repush yet.

Reply:
This works:
```
SNAME_FILE=${BATS_FILE_TMPDIR}/services
```
(Will need some editing in order to work on RHEL; I will start on that process ASAP). Please also add this in teardown:
```diff
             rm -f $UNIT_DIR/$line.{service,timer}
         done < $SNAME_FILE
+        rm -f $SNAME_FILE
```
...and please add this to the end of your new test:
```diff
     run_podman rm -f -t0 --ignore $ctr
+    run_podman rmi $image_on_local_registry
```

Reply:
Thank you, Ed!  I am sitting tight until I have green light.  Note that we don´t need to rush.  There's still time to get this fix in.  And if it's a .1 fix, it's not a problem.